src/back/services/notification.ts

```diff
@@ -277,17 +284,117 @@ export class NotificationService {
     })
   }
 
-  static authoredProposalFinished(proposal: ProposalAttributes) {
+  static async pitchPassed(proposal: ProposalAttributes, addresses: string[]) {
     inBackground(async () => {
       try {
-        const coauthors = await CoauthorService.getAllFromProposalId(proposal.id)
-        const coauthorsAddresses = coauthors.length > 0 ? coauthors.map((coauthor) => coauthor.address) : []
-        const addresses = [proposal.user, ...coauthorsAddresses]
+        if (!areValidAddresses(addresses)) {
+          throw new Error('Invalid addresses')
+        }
+
+        const title = Notifications.PitchPassed.title(proposal)
+        const body = Notifications.PitchPassed.body
+
+        const validatedUsers = await UserModel.getActiveDiscordIds(addresses)
```

Comment:
I think this could be inside the `DiscordService`, since we always need to get the active discord ids, and the notification service only needs them to hand them out.

```typescript
static async sendDirectMessages(addresses, message){
   const validatedUsers = await UserModel.getActiveDiscordIds(addresses)
   for (const user of validatedUsers) {
      this.sendDirectMessage(user.discord_id, {
        title,
        action: body,
        url: proposalUrl(proposal.id),
        fields: [],
      })
    }
```

WDYT?